tests/cube/api/editoverview.api.test.ts

```diff
@@ -16,6 +16,7 @@ jest.mock('../../../src/dynamo/models/cubeHash');
 describe('Edit overview API', () => {
   afterEach(() => {
     jest.clearAllMocks();
+    jest.resetAllMocks();
```

Comment:
Weirdly without both, things like mockRejectedValue were persisting between tests. Caused me lots of confusion!